icons/horusec.svg

```diff
@@ -0,0 +1 @@
4-1.737.089.48.215.948.384 1.404.317.82.424.983.977 1.901.61.91 1.403 1.772 2.39 2.594 2.273 1.894 4.964 3.305 8.225 4.312l.416.128.416-.128c3.33-1.029 6.042-2.44 8.29-4.312 2.71-2.258 3.984-4.819 3.898-7.83-.008-.283-.015-4.26-.02-7.73a9.68 9.68 0 01-.407.792c-.37.633-.94 1.278-1.606 1.895z" fill="url(#prefix__paint0_linear)"/><path d="M27.996 3v.928l.001.076v.641l.001.24v.1L28 6.618l-.013.033c-.14.367-.274.717-.299.772l-.052.118c-.176.393-.36.795-.58 1.169l-.043.073c-.357.585-.874 1.166-1.451 1.677l-.066.061a8.17 8.17 0 01-.134.122l-.074.065c-.672.587-1.39 1.127-2.244 1.525l-.143.067c-.333.157-.664.317-1.008.454l-.026.01c-.334.133-.36.135-.362.432V22.169l-.041.03a26.497 26.497 0 01-2.302 1.49v-.6V21.667v-1.154-.874-.767-.638-.627-.485-.118l-.001-.347v-.441-.409V15.523 15.18 14.952v-.2V14.582l-.001-.05v-.121c-.003-.13-.018-.264-.03-.45l-.047.016c-.167.061-.293.1-.409.155l-.859.414c-.572.277-1.144.555-1.718.83l-.88.419c-.881.419-1.764.836-2.643 1.259l-.008.004c-.227.109-1.438.697-2.938 1.426l-.002-.008-.08.043c-.24.124-.496.23-.765.313l-.008.004H8.77l-.08.023a5.013 5.013 0 01-1.35.198h-.086l-.091-.001a5.077 5.077 0 01-.716-.068l-.057-.01-.057-.011-.095-.02a5.673 5.673 0 01-.464-.121l-.057-.018-.056-.019.001.002-.052-.018C4.037 18.238 3 16.974 3 15.465v-.064l.006-.128v-.06h.006l.004-.052c.13-1.299 1.084-2.401 2.423-2.938l.068-.027a4.977 4.977 0 011.727-.322l.027-.001h.093c1.4.018 2.638.598 3.405 1.484l.032.03c.073.072.134.153.181.24l.036.05.034.052-.019.008.013.036a.857.857 0 01.042.267v.029c-.018.544-.551.981-1.206.981h-.04c-.405-.011-.758-.19-.967-.455l-.012.005-.027-.033c-.276-.331-.7-.568-1.191-.646l-.057-.008a2.2 2.2 0 00-.29-.02h-.043c-.914.018-1.664.583-1.806 1.32l-.01.05c-.005.035-.008.07-.011.105l-.001.175.002.035c.07.815.878 1.459 1.869 1.459l.063-.001c.187-.006.367-.035.537-.083v.001l.007-.003.008-.003c.34-.121 7.005-3.26 7.403-3.45l-.008-.003c-.218-.079-2.058-.56-2.809-.883l-.107-.047c-1.036-.457-1.97-1.026-2.785-1.742l-.085-.076A9.43 9.43 0 017.797 8.74l-.015-.021c-.105-.156-.245-.23-.482-.228h-.02c-.151.002-.425.002-.718.003h-.693l-.035.001H5.409V10.031c-.018.535-.542.965-1.185.965h-.07c-.64-.016-1.152-.455-1.152-.994V3h24.994zm-2.408 2.018H5.409V6.53h.035v.004h3.402c.35 0 .35.019.494.329l.035.073a7.592 7.592 0 003.083 3.374l.097.056c.645.368 1.331.646 2.065.872l.097.029c1.033.304 2.091.393 3.172.348l.107-.005a8.944 8.944 0 002.094-.361l.12-.037c.558-.172 1.09-.376 1.6-.632l.095-.049a8.282 8.282 0 001.687-1.18l.07-.062c.508-.461.943-.97 1.268-1.538l.066-.114c.196-.343.39-.68.586-1.054v-.026l.006-1.54zM17.351 6.6h.29c2.32.002 4.553.022 4.745.06l-.023.068c-.208.58-.56 1.093-1.029 1.544l-.05.048c-.967.906-2.204 1.415-3.643 1.5l-.078.005c-2.58.12-4.902-1.439-5.304-3.165.144-.035 2.073-.054 4.21-.059H17.35z" fill="#616161"/><defs><linearGradient id="prefix__paint0_linear" x1="28" y1="12.7" x2="7.489" y2="26.285" gradientUnits="userSpaceOnUse"><stop stop-color="#F7941E"/><stop offset="1" stop-color="#EF4123"/></linearGradient></defs></svg>
```

Comment:
The icon looks somehow stretched. Please use the correct ratio as shown on their website. In addition, it would be better if the color would be a single color for the whole SVG. Please try to colorize the logo with a nice - probably orange - color:

![image](https://user-images.githubusercontent.com/12248527/121598516-2c9bea80-ca42-11eb-9477-c7433d1c58bf.png)

You can use this color picker, to select a proper color for it:
https://pkief.github.io/material-color-converter/

Reply:
All good! I made the corrections in size and also in color, I used the tool that informed me, to generate an orange material similar to our logo.

Reply:
Any more changes needed?

Reply:
I adjusted the size and color of the icon.